### ISSUE TITLE
An Option Page for configuration variable could be very useful

### ISSUE THREAD

**Issue by vtasio** [Patrowl/PatrowlManager#196]
A page that configuration such as the below could be saved/changed and used:

- Scan Engine Max retries
- Number of second to wait before perform a request to scan engine
- In how many items to split the assets during scan
- Haw many concurrent jobs to start
- etc...

**Comment by MaKyOtOx** [comment]
hi @vtasio : great idea. We could also use the Settings model for global configuration parameters ?

**Comment by vtasio** [comment]
To tell you the truth , i have not used that menu at all. But i think i got it . You can use in setting all the variable that can be found in https://github.com/Patrowl/PatrowlDocs/blob/master/installation/patrowlmanager-settings.md and override the one's in the confine files ?

**Comment by MaKyOtOx** [comment]
Yes, it would be nice. We just have to clearly specify a priority order. Example:
Scan Settings > Engine Settings > Global Settings > Environment variables

**Comment by MaKyOtOx** [comment]
Yes, I'm just using the Settings model for notification settings. BTW, it could be extended to all global variables ?

**Comment by vtasio** [comment]
I will try to find a way to make but, this might be a little more complicated for my knowledge. If you start with something that i can rely on  (before me ), i believe i could extend it .

**Comment by vtasio** [comment]
Question: are you setting up somewhere the number of Jobs to it is the outcome of asset split during scan ? for example 400 assets scan , splited by 100 . This will produce 4 concurrent jobs ?
